torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -612,4 +618,8 @@ def functional_call(*args, **kwargs):
             )
         return out
 
+    # This is fine in terms of perf because we only pass this around for export path
```

Comment:
Can you add more detail to this comment:
```
Note [Preserving the nn module stack metadata during export non-strict mode]
This path is currently only used by the non-strict export flow, where we cannot rely on dynamo to preserve nn stack metadata in our captured graph. Instead, we stash the original user nn module here, and rely on `make_fx` to grab this stashed module and use it to track nn module stack metadata
```

And then also mention this comment directly in `proxy_tensor.py`, where we rely on this invariant being set:
```
See Note [Preserving the nn module stack metadata during export non-strict mode]
```